src/resources/views/crud/fields/icon_picker.blade.php

```diff
@@ -1,7 +1,7 @@
 <!-- icon picker input -->
 @php
-    // if no iconset was provided, set the default iconset to Font-Awesome
-    $field['iconset'] = $field['iconset'] ?? 'fontawesome';
+    // if no iconset was provided, set the default iconset to Line-Awesome
```

Comment:
I think we should not change default iconset. It would break people apps because they don't have the js needed to make line work.

Reply:
Agree, we should keep `fontawesome`

Reply:
```suggestion
    // if no iconset was provided, set the default iconset to Font-Awesome
```